tests/tests/test_github_webhooks.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import os
+import requests
+import time
+
+BASE_DIR = os.path.dirname(__file__)
+
+
+def load_payload(filename):
+    with open(os.path.join(BASE_DIR, "payloads", filename), "rb") as f:
+        return f.read()
+
+
+def test_pull_request_opened(integration_test_runner_url):
+    res = requests.post(
+        integration_test_runner_url + "/",
+        data=load_payload("pull_request_opened.json"),
+        headers={
+            "Content-Type": "application/json",
+            "X-Github-Event": "pull_request",
+            "X-Github-Delivery": "delivery",
+        },
+    )
+    assert res.status_code == 200
+    #
```

Comment:
?

Reply:
It's a separator, I don't like leaving empty lines in Python code